docs/product/performance/assets.mdx

```diff
@@ -118,15 +118,15 @@ The following tokens will be replaced with \* within a resource url
 
 <Note>
 
-Resource parametrization is still a work-in-progress. As these improvements are made, you will will temporarily see instances of the new and old groupings in your Resource Module.
+Asset parametrization is still a work-in-progress. As these improvements are made, you will will temporarily see instances of the new and old groupings in your Asset Module.
 
 Let us know of any feedback through Github Issues.
 
 </Note>
 
-To see an example of resource URL from a group, hover over a URL in the resource table.
+To see an example of asset URL from a group, hover over a URL in the asset table.
 
-## Resource Summary Page
+## Asset Summary Page
 
 To open a resource's **Resource Summary** page, click on the resource from the table in the **Resources** page.
```

Comment:
This still mentions resource instead of assets, did you mean to replace with assets for the rest of the page too?

Reply:
Yeah this should be assets, good catch! , I'll have to go through this to double it all. There are some cases we need to keep the term Resources (like mentioning span ops), so I cant do a replaceall